generators/createtodoapp/templates/infra/core/host/container-app.bicep

```diff
@@ -124,6 +127,11 @@ resource app 'Microsoft.App/containerApps@2023-05-02-preview' = {
         appProtocol: daprAppProtocol
         appPort: ingressEnabled ? targetPort : 0
       } : { enabled: false }
+      runtime: isJava ? {
+        java: {
+          enableMetrics: true
```

Comment:
@hangwan97 do we need to enable the dynamic log level change related feature here?

Reply:
try use java: {} and use the default value for service?

Reply:
I would vote for explicitly specify the values rather than hidden the settings in defaults.